docs/source/experiments/regression-robust04.md

```diff
@@ -0,0 +1,86 @@
+# PISA: Regression Tests for [Disks 4 & 5](https://trec.nist.gov/data_disks.html) (Robust04)
+
+## Indexing
+
+First, we will create a directory where all the indexes are going to be stored:
+
+```
+mkdir robust04
+```
+
+### Parsing
+
+```
+gzip -dc $(find /path/to/disk45/ -type f -name '*.*z' \( -path '*/disk4/fr94/[0-9]*/*' -o -path '*/disk4/ft/ft*' -o -path '*/disk5/fbis/fb*' -o -path '*/disk5/latimes/la*' \)) | bin/parse_collection -f trectext -b 10000 --stemmer porter2 --content-parser html -o robust04/fwd
+```
+
+You can replace `gzip -dc` with `zcat` on Linux or `gzcat` on MacOS.
+The directory `/path/to/disk45/` should be the root directory of [TREC Disks 4 & 5](https://trec.nist.gov/data_disks.html).
+
+### Inverting
+
+```
+/path/to/pisa/build/bin/invert -i robust04/fwd -o robust04/inv -b 400000 --term-count $(cat robust04/fwd.terms | wc -l)
+```
+
+### Reordering
+We perform Recursive Graph Bisection (aka BP) algorithm, which is currently the state-of-the-art for minimizing the compressed space used by an inverted index (or graph) through document reordering. 
+
+```
+/path/to/pisa/build/bin/recursive_graph_bisection -c robust04/inv -o robust04/inv.bp --documents robust04/fwd.doclex --reordered-documents robust04/fwd.bp.doclex
+```
+
+### Meta data
+To perform BM25 queries it is necessary to build an additional file containing the parameters needed to compute the score, such as the document lengths. The file can be built with the following command:
```

Comment:
I would say "information" instead of "parameters" and at the end of the sentence add: "such as the document lengths, **max scores, or block max structure.**"

Reply:
Maybe let's also change the last sentence to something like: _The following command builds a metadata file with block-max structure with blocks of fixed size of 64 postings:_

Reply:
`max scores, or block max structure` are performance features which are not really needed here since we are measuring only about effectiveness. Also, they are not required by BM25